app/models/document.rb

```diff
@@ -31,6 +31,8 @@ class Document < ApplicationRecord
 
   delegate :topics, to: :document_topics
 
+  self.ignored_columns = %w(document_type_id)
```

Comment:
Nice! I don't quite see the benefit of it here though - why are we hiding the column? Why are we waiting until the next PR to remove it? Maybe a bit more in the commit message here would be useful.

Reply:
Heya, have adjusted the commit message to better reflect why this is going on. Fully expect this code to be removed but for the time being it gives us a bit of peace of mind that our changes won't break stuff. Have kept it as a separate commit to make sure that we remember why this is happening / that it can be removed in the future.

Reply:
Thanks 👍